apps/workers/deduper/worker.go

```diff
@@ -98,6 +98,9 @@ func newWorker(info string) task.Worker {
 	stsFiles := make(StatsFiles, len(fSts))
 
 	for i, sts := range fSts {
+		if sts.IsDir {
+			continue
+		}
```

Comment:
keep current behaviour by skipping directories

Reply:
Before this change, would there be a problem if there was a directory in the folder?

Reply:
no, directories were just being ignored at a lower level since the stat object didn't used to have a isDir field in it.